frontend/src/slices/prompts/prompts.slice.ts

```diff
@@ -1,18 +1,23 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, isAnyOf } from '@reduxjs/toolkit';
 
 import { DataStatus } from '~/libs/enums/enums.js';
 import { type ValueOf } from '~/libs/types/types.js';
-import { type GenerateArticlePromptResponseDto as Prompt } from '~/packages/prompts/prompts.js';
+import {
+  type GenerateArticlePromptResponseDto as generatedPrompt,
+  type PromptBaseResponseDto as createdPrompt,
+} from '~/packages/prompts/prompts.js';
```

Comment:
Maybe it will be better to rename types to UpperCase to be consistent with types naming